src/software/embedded/robot_diagnostics_cli/robot_diagnostics_cli.py

```diff
@@ -123,56 +123,41 @@ def __generate_stats_table(self) -> Table:
         )
         return table
 
-    def __generate_redis_table(self) -> Table:
+    def __generate_yaml_table(self) -> Table:
         """Make a new table with embedded_data.redis value information."""
         table = Table(show_header=True, header_style="bold blue")
         table.add_column("Redis Value Name")
         table.add_column("Key", style="dim")
         table.add_column("Value")
 
         table.add_row(
-            "Robot ID", f"{ROBOT_ID_REDIS_KEY}", self.embedded_data.get_robot_id()
+            "Robot ID", f"{ROBOT_ID_YAML_KEY}", self.embedded_data.get_robot_id()
         )
         table.add_row(
             "Channel ID",
-            f"{ROBOT_MULTICAST_CHANNEL_REDIS_KEY}",
+            f"{ROBOT_MULTICAST_CHANNEL_YAML_KEY}",
             self.embedded_data.get_channel_id(),
         )
         table.add_row(
             "Network Interface",
-            f"{ROBOT_NETWORK_INTERFACE_REDIS_KEY}",
+            f"{ROBOT_NETWORK_INTERFACE_YAML_KEY}",
             self.embedded_data.get_network_interface(),
         )
         table.add_row(
             "Kick Constant",
-            f"{ROBOT_KICK_CONSTANT_REDIS_KEY}",
+            f"{ROBOT_KICK_CONSTANT_YAML_KEY}",
             self.embedded_data.get_kick_constant(),
         )
         table.add_row(
             "Kick Coefficient",
-            f"{ROBOT_KICK_EXP_COEFF_REDIS_KEY}",
+            f"{ROBOT_KICK_EXP_COEFF_YAML_KEY}",
             self.embedded_data.get_kick_coeff(),
         )
         table.add_row(
             "Chip Pulse Width",
-            f"{ROBOT_CHIP_PULSE_WIDTH_REDIS_KEY}",
+            f"{ROBOT_CHIP_PULSE_WIDTH_YAML_KEY}",
             self.embedded_data.get_chip_pulse_width(),
         )
-        table.add_row(
-            "Battery Voltage",
-            f"{ROBOT_BATTERY_VOLTAGE_REDIS_KEY}",
-            self.embedded_data.get_battery_volt(),
-        )
-        table.add_row(
-            "Battery Current Draw",
-            f"{ROBOT_CURRENT_DRAW_REDIS_KEY}",
-            self.embedded_data.get_current_draw(),
-        )
-        table.add_row(
-            "Capacitor Voltage",
-            f"{ROBOT_CAPACITOR_VOLTAGE_REDIS_KEY}",
-            self.embedded_data.get_cap_volt(),
-        )
```

Comment:
`RobotStatus` has current draw and cap voltage, you can move these two fields to the `__generate_stats_table` function. You can pipe this data into `embedded_data` through this listener: https://github.com/UBC-Thunderbots/Software/blob/3c8ee82dcdc60339afa84646d5b1c62330d92bed/src/software/embedded/robot_diagnostics_cli/embedded_communication.py#L54